src/components/title/_forms/add-title-to-package-form.js

```diff
@@ -0,0 +1,21 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+
+import PackageSelectField from '../_fields/package-select';
+import CustomURLField from '../_fields/custom-url';
+
+function AddTitleToPackageForm({ packageOptions }) {
```

Comment:
Is this still accurate as a `*Form` with these changes? If this becomes a common pattern, would something like `*Fieldset` better describe these field groups that don't actually belong to forms yet?

Reply:
The `react-final-form` docs call this concept a "field group", so I changed to that nomenclature.